layout/layout.ejs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="<%= config.language %>">
+<html<% if(config.language) { %>lang="<%= config.language %>"<% } %>>
```

Comment:
I have a feeling this will render

`<htmllang="en">`

Did you test it ? Can paste the rendered code in both cases (language and no language set)

thanks